tests/test_infrastructure_cache.py

```
@@ -0,0 +1,101 @@
+import pytest
+import asyncio
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)